### DISCUSSION TITLE
0.11.0 Preview 01

### DISCUSSION THREAD

**Discussion by KrisCris** [KrisCris/Palworld-Pal-Editor#75] [General]
Fixed save loading issue introduced in 0.4.11?

Please backup your save, this is highly experimental.

Duplicated topic will be collapsed.

<hr /><em>This discussion was created from the release <a href='https://github.com/KrisCris/Palworld-Pal-Editor/releases/tag/0.11.0_preview01'>0.11.0 Preview 01</a>.</em>

**Comment by stephenshua** [comment]
nice!wait so long time like three thousand years!

**Comment by deng1262613347** [comment]
nice, now it works,
but lack new pals ,new passives and new attacks,
looking forward to the new version.

**Comment by andyvalci** [comment]
error
![Captura de tela 2024-12-24 115313](https://github.com/user-attachments/assets/81f77e3c-a770-4c7f-a834-a5dda7d77fab)

**Reply by KrisCris** [reply]
which editor version are u using? this error should have already been fixed long ago

**Reply by andyvalci** [reply]
> which editor version are u using? this error should have already been fixed long ago

I managed to get the editor to work, I was trying to edit a save that I started in random mode, as the editor still doesn't have the base date with the game's new pals, it will even work, but in addition to not having the new pals, you won't either you will earn boss medals every time you beat them, new content, that is, everything new content ends up buggy when I use the editor, looking forward to fixing the bugs

**Reply by KrisCris** [reply]
What is random mode

**Reply by andyvalci** [reply]
It's the new mode that the game brought, the mod is unfeasible to use because the new things in the game don't appear in it, like the new pals for example, that's the editor's problem, it doesn't have the new game database in it.

**Reply by KrisCris** [reply]
New data will probably come later. Don’t have the time to do it yet.

**Reply by KrisCris** [reply]
OK, the uint32 issue is fixed, try if the latest version works for you.

**Comment by shiyanxiaogou** [comment]
11版本好像还没有新的词条技能

**Comment by lch1164840694** [comment]
![1735106926302](https://github.com/user-attachments/assets/b52096fa-a57b-4aaa-a5cc-a950def3958a)
打开以后出现这个无法进入是我电脑出现了什么问题吗还是缺少那些必备的软件啊

**Reply by KrisCris** [reply]
https://github.com/KrisCris/Palworld-Pal-Editor/issues/4

**Reply by lch1164840694** [reply]
感谢！！！

**Comment by shiyanxiaogou** [comment]
很高兴收到您的回复，我用的是11版本的预览版，但是他似乎没有更新出最新版本的帕鲁词条，我被这款游戏的宠物词条深深困扰，因此看到您的帖子我深感欢喜，但是我看到了其中的不足，其一，背包中的宠物似乎看不到，其二，新版本的词条似乎是没有更新出来，可能是我认不出英文的原因，但是中文版没有翻译出来是真的，第三，帕鲁可以用帕鲁之魂加强，但是修改器里面似乎没有这个功能，
最后感谢您对这款游戏的付出。






 ----------Reply to Message----------
 On Wed, Dec 25, 2024 14:00 PM ***@***.***&gt; wrote:

  

  
which editor version are u using? this error should have already been fixed long ago
  
I managed to get the editor to work, I was trying to edit a save that I started in random mode, as the editor still doesn't have the base date with the game's new pals, it will even work, but in addition to not having the new pals, you won't either you will earn boss medals every time you beat them, new content, that is, everything new content ends up buggy when I use the editor, looking forward to fixing the bugs

**Reply by KrisCris** [reply]
You are replying to an unrelated thread, lol.
词条以后会慢慢加，现在没时间

**Comment by adc1ess** [comment]
![image](https://github.com/user-attachments/assets/adb9fd69-b758-442d-9a04-ea903c88f4fe)
it seems that the new pal not add or it may be my Incorrect installation

**Comment by ken0528** [comment]
![image](https://github.com/user-attachments/assets/817ccedd-1f5a-4640-b4b3-35c8d5e8e8d9)
